singularity-eos/eos/eos.hpp

```diff
@@ -528,9 +529,12 @@ class SpinerEOSDependsRhoT : public EosBase<SpinerEOSDependsRhoT> {
   }
   PORTABLE_FORCEINLINE_FUNCTION Real MinimumDensity() const { return rhoMin(); }
   PORTABLE_FORCEINLINE_FUNCTION Real MinimumTemperature() const { return T_(lTMin_); }
+  PORTABLE_INLINE_FUNCTION
   int nlambda() const noexcept { return _n_lambda; }
-  inline RootFinding1D::Status rootStatus() const { return status_; }
-  inline TableStatus tableStatus() const { return whereAmI_; }
+  PORTABLE_INLINE_FUNCTION
+  RootFinding1D::Status rootStatus() const { return status_; }
+  PORTABLE_INLINE_FUNCTION
+  TableStatus tableStatus() const { return whereAmI_; }
```

Comment:
Making these `PORTABLE_INLINE_FUNCTION` won't do any harm. It's worth noting though that when these functions are called in a parallel context, it's undefined behavior.